static/js/components/dashboard/CourseAction.js

```diff
@@ -192,6 +208,12 @@ export default class CourseAction extends React.Component {
       action = this.renderEnrollButton(run);
       description = this.renderTextDescription('Processing...');
       break;
+    case STATUS_PAID_BUT_NOT_ENROLLED:
+      description = this.renderContactUsDescription(
+        'Something went wrong.\n You paid for this course but \nare not enrolled.',
```

Comment:
This copy text is out of date. Can you use the text in the screenshot on the issue? It looks like we don't need the email address either